packages/graphql-client/documents/delegateDelegationHistory.graphql

```diff
@@ -0,0 +1,42 @@
+query GetDelegateDelegationHistory(
```

Comment:
can't you use the same delegationHistory used on the holder delegation histories?